index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header>
+        <div class="logo"><a href="">Johnson</a> </div>
+        <nav> <a href=""><img class="image" src="images/Union.png" alt="dropdown"></a> </nav>
+    </header>
+    <main>
+        <section class="headline">
+            <div class="supporting-text">
+                <h2>Hey There. I'm Johnson</h2>
+                <p> I'm a software developer <br /></p> 
+                <div class="description">
+                    I can help you build a product , feature or website Look through
+                    some of my work and experience! If you like what you see and have a project you need coded,
+                    don't hestiate to contact me.
+
+                </div>
+
+                
+            </div>
+            <nav class="socialmedia">
+                <a href=""><img src="images/Disabled.png" alt=""></a>
+                <a href=""><img src="images/Enabled.png" alt=""></a>
+                <a href=""><img src="images/GitHub.png" alt=""></a>
+                <a href=""><img src="images/Twitter.png" alt=""></a>
+                <a href=""><img src="images/micro.png" alt=""></a>
+            </nav>
+            
+                <img class="illustration" src="images/Illustration.png" alt="">
+            
+
```

Comment:
- [ ] Kindly make sure this image is used as a background image in the **home section** in the `style.css` file and not as `<img>` tag in the HTML file and style it correctly as from the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27) design as required in the project. i.e you should remove this `<img>` **tag** from HTML and use the image in the CSS fils as a background image.